package/src/components/Attachment/Gallery.tsx

```diff
@@ -158,14 +160,14 @@ const GalleryWithContext = <
     minHeight,
     minWidth,
   };
-
+  const imagesAndVideos = [...images, ...videos];
```

Comment:
Overall looks good. Just have one doubt. Don't you think the order in which the attachments were added will be changed in the gallery where you get images first and then the videos?

Reply:
I don't think this would be a problem so gtg 😄

Reply:
Probably later we can add some ordering to them if someone requests

Reply:
Yes, that is why I used `imagesAndVideos` as one source but yes to avoid a major release this makes sense. 😄